installFeatures.ps1

```diff
@@ -0,0 +1,24 @@
+# Define the path to the CSV file containing the list of features
+$featuresFilePath = "C:\path\to\features.csv"
```

Comment:
Seems like a generic path that won't exist anywhere.
Can you find a better way to get to the file?

Reply:
I used s3 as stored location